.github/ISSUE_TEMPLATE/platform-onboarding-checklist.md

```diff
@@ -67,17 +67,26 @@ RA - Remote Access_
 
 - [ ] GitHub
   - [ ] Join the ['department-of-veterans-affairs' organization](https://github.com/department-of-veterans-affairs)
-  - [ ] Even though you're Platform and not VFS, fill out the [New VFS Team Member Template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?template=New-VFS-Team-Member.yml). By filling that out, you'll get added to the `vets.gov-write` GitHub Team which grants you write access to most repos in the org.
+  - [ ] Even though you're Platform and not VFS, fill out the [New VFS Team Member Template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?template=New-VFS-Team-Member.yml). By filling that out, you'll get added to the `vets-gov-write` GitHub Team which grants you write access to most repos in the org.
   - [ ] Make sure you have access to the [devops repo](https://github.com/department-of-veterans-affairs/devops)
+  - [ ] [GitHub Copilot](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?template=github-copilot-onboarding.yml) - _AI pair programming assistant, available after GitHub org access_
```

Comment:
This is linked on the "Request access to tools" page https://depo-platform-documentation.scrollhelp.site/getting-started/request-access-to-tools#Requestaccesstotools-GettingCopilotaccess. Not sure if we need it here as well. But not opposed to it.